src/index.js

```diff
@@ -76,92 +79,295 @@ class Radar {
       return;
     }
 
-    if (!navigator || !navigator.geolocation) {
+    Navigator.getCurrentPosition((status, { accuracy, latitude, longitude }) => {
+      if (status != STATUS.SUCCESS) {
+        callback(status);
+      }
+
+      // Get user data
+      const deviceId = Device.getId();
+      const userId = Cookie.getCookie(Cookie.USER_ID);
+      const placesProvider = Cookie.getCookie(Cookie.PLACES_PROVIDER);
+      const description = Cookie.getCookie(Cookie.DESCRIPTION);
+      const _id = userId || deviceId;
+
+      // Setup http
+      const headers = {
+        Authorization: publishableKey
+      };
+
+      const body = {
+        accuracy,
+        description,
+        deviceId,
+        deviceType: 'Web',
+        foreground: true,
+        latitude,
+        longitude,
+        placesProvider,
+        sdkVersion: SDK_VERSION,
+        stopped: true,
+        userAgent: navigator.userAgent,
+        userId,
+      };
+
+      const host = Cookie.getCookie(Cookie.HOST) || DEFAULT_HOST;
+      const url = `${host}/v1/users/${_id}`;
+      const method = 'PUT';
+
+      const onSuccess = (response) => {
+        try {
+          response = JSON.parse(response);
+          if (callback) {
+            callback(STATUS.SUCCESS, position.coords, response.user, response.events);
```

Comment:
I don't think `position` exists here.

Reply:
good catch, changing to `{ accuracy, latitude, longitude }`